src/Orchard.Web/Modules/Orchard.Setup/Services/SetupService.cs

```diff
@@ -154,6 +155,10 @@ public class SetupService : ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
 
+                    // Create the distributed lock record schema.
+                    var distributedLockSchemaBuilder = new DistributedLockSchemaBuilder(_shellSettings, schemaBuilder);

```

Comment:
Look at this line:
https://github.com/OrchardCMS/Orchard/pull/5752/files#diff-c4d675a4ef668ca2d924df563f3d2548R41

Then if you see it you understand you can __remove__ the calls to the same thing from both Setup classes.